### ISSUE TITLE
fix(sideNav):keyboard navigation

### ISSUE THREAD

**Issue by sstrubberg** [carbon-design-system/gatsby-theme-carbon#754]
When I tab through the page, there is no visual keyboard indication on the following:

all of the left navigation elements
Privacy
Terms of use
IBM.com/design
Medium
Twitter
To Reproduce

For the links in the left navigation:

Go to https://w3.ibm.com/design/
Press the tab key 4 times. Keyboard focus is actually on "Resources" but you can't see that.
Press Enter key and you'll see the sub-navigation drop open...but with no keyboard focus shown (focus is actually still on "Resources") because if you press Enter again, the sub-navigation closes again. The same goes for all other items.
The only special visual treatment I see is when Enter is pressed while focus is on one of the sub-topic items. This is a selection state, not visual keyboard focus because you can Tab away from that item and the selection state remains (as it should) but no visual keyboard focus treatment is ever shown in the navigation.
For the links in the footer:

Go to https://w3.ibm.com/design/
Tab through the page till you get toward the bottom. You'll see the focus indication on Next: Resources: Design practices
The next Tab stops are stopping somewhere, but you can't tell where. There's no focus indicator.
If you press enter, you'll go to one of the above listed items.
Expected behavior
Should have a visible keyboard focus indicator when focus is on each of those elements.

Screenshots
Privacy has keyboard focus - but you can't tell.
Screen Shot 2020-01-31 at 1 17 57 PM

Desktop (please complete the following information):

OS: macOS
Browser: Safari
Version: 13.0.4 (14608.4.9.1.4)
